src/core/core/service/news_item_tag.py

```diff
@@ -68,24 +68,25 @@ def get_n_biggest_tags_by_type(cls, tag_type: str, n: int, days: int = 0) -> lis
         return [{"name": row[0], "size": row[1]} for row in result]
 
     @classmethod
-    def get_tag_types(cls) -> list[tuple[str, int]]:
```

Comment:
It wasn't available, since I filter based on the number of "days" passed in as a variable. The count before was global and not reflecting the filtered state. Therefore, only the "new" calculation yields the correct value.